pyuvdata/utils.py

```diff
@@ -1081,9 +1081,64 @@ def get_lst_for_time(jd_array, latitude, longitude, altitude):
     return lst_array
 
 
-def find_clusters(location_ids, location_vectors, tol):
+def _adj_list(vecs, tol, n_blocks=None):
```

Comment:
Do you want to plumb the `n_blocks` parameter up to the `find_clusters` function? If not, I'm not sure we should have it in the function signature here.

Reply:
The n_blocks keyword here is used by unit tests. Otherwise, the case of `n_blocks > 1` won't get tested.

Reply:
Ok. But if it's not accessible from the code users will call, what's the point in having it as an option in the helper function?

Reply:
The test calls this function directly, and I don't know of another way to set the parameter except by making it a keyword. It shouldn't be used by end users.

Reply:
No, I'm asking why is this an option in the helper function, and why there's an if statement for `n_blocks` being specified if no one can actually specify that. In other words, why handle a user specifying `n_blocks` if they actually cannot specify it.

Reply:
The users can't specify it, but it does get specified by a test. So the keyword is just there for the test.

Normally, `n_blocks` is only larger than 1 if the len(vecs) > 2**10. I wanted to test that the function works correctly when the vector is is broken up into blocks, and for that to happen I either need to artificially set the number of blocks or make a very large `vecs`. I included this keyword just for that test.

I guess it wouldn't hurt to make it available to users.

Reply:
Ah, ok. I was trying to understand what the value was, but a way to test that more than one block works without making the test take forever is a good reason. I'm ok with it as is then, I just didn't understand the purpose.